modules/common/condition/funcs.go

```diff
@@ -157,6 +157,13 @@ func (conditions *Conditions) Sort() {
 	})
 }
 
+// SortByLastTransitionTime - Sorts a list of conditions by the LastTransitionTime
```

Comment:
Is the sorting here descending or ascending?  `lessLastTransitionTime` function makes it seem like it is descending, but just want to be sure

Reply:
Read a bit further.  It is descending: `// get the first conditon of the group which is the one with the latest LastTransitionTime`